src/design/typography/demo/figure.twig

```diff
@@ -0,0 +1,4 @@
+<figure>
+  <img src="https://picsum.photos/600/300">
```

Comment:
Nit: Can you add a `width`, `height` and `alt` just so this feels in line with our best practices?

Reply:
Another thought: Could we pull an image on our blog to use as FPO for this?

I believe you can add the image to the `demo` directory, then in the `.mdx` file you can say `import 'path/to/image.svg` (or whatever) and it _should_ build with Webpack.